src/python/artm/library.py

```diff
@@ -564,7 +611,7 @@ def GetRegularizerState(self, regularizer_name):
         regularizer_state.ParseFromString(state_blob)
         return regularizer_state
 
-    def GetThetaMatrix(self, model=None, batch=None, clean_cache=None, args=None, topic_names=None):
+    def GetThetaMatrix(self, model=None, batch=None, clean_cache=None, args=None, topic_names=None, use_matrix=True):
```

Comment:
The idea was to promote this change as much as possible. I'm not 100% sure whether this is a good idea, but pros are that every BigARTM user (in python) will learn that now there is a new option of getting phi/theta as a numpy matrix. If they don't like it, it takes 30 seconds to fix their script by adding "use_matrix = False" (and that will be bold-highlighted in release notes). But I'm sure everyone will like it - it is more elegant and works substantially faster.

Typically I don't want to force users to upgrade, but this case sounds a bit special.